tests/HtmlProviderTest.php

```diff
@@ -37,7 +37,7 @@ public function testAllowSomeExternalImages()
         $this->assertEmbed(
             'http://www.brothers-brick.com/2016/04/06/stunning-lego-darth-vader-mask-cleverly-hides-scenes-from-star-wars/',
             [
-                'images' => 3,
+                'images' => 1,
```

Comment:
In theory, there would be at least 2 images here because this url contains 2 images from flickr, that should be allowed.

Reply:
After looking at this website, it should be the `2` mages, not `1`.